src/server/game/Spells/SpellEffects.cpp

```diff
@@ -3972,6 +3972,11 @@ void Spell::EffectScriptEffect(SpellEffIndex effIndex)
                             m_caster->CastSpell(m_caster, 61268, true);
                             return;
                         }
+                    case 62536: // Removes Warts-B-Gone Lip Balm
+                        {
+                            m_caster->RemoveAurasDueToSpell(62574);
+                        }
+                        break;
```

Comment:
Oh no, don't add more of those here, create a script for the spell instead

Reply:
But I like it there :(

Reply:
It's still here >:(